### PR TITLE
Adding MirrorMaker examples

### PR DESCRIPTION
**Pull request by davidespo**
Added two examples for circular replication across three global regions, and an Active<=>Active configuration between two regions.

### CODE REVIEW

**Comment by davidespo** [comment]
Rendered READMEs:
https://github.com/aiven/terraform-provider-aiven/tree/mm2/examples/kafka_mirrormaker_bidirectional
https://github.com/aiven/terraform-provider-aiven/tree/mm2/examples/kafka_mirrormaker_global

File: docs/index.md [hunk 1]
```diff
@@ -29,32 +29,39 @@ attempted to be deleted so even with this setting enabled you need to be very ca
 with the changes that are to be applied.
 
 ## Installation instructions for Terraform client 0.12
-*If you can please upgrade your Terraform client to version 0.13 or above, in this case, 
-there is no need to install provide manually.*
 
-Download the Aiven provider for Linux AMD64. 
+_If you can please upgrade your Terraform client to version 0.13 or above, in this case,
+there is no need to install provide manually._
```
Comment (encima) [inline docs/index.md:34]: *provider

**Review by encima — DISMISSED** [review]
Nice! Arch diagram is super useful as well. Would be nice to have a way to auto update the index with these examples somehow.

I am not sure if the clean.sh script is needed (partly because newcomers to terraform may think it is then ok to remove the state prematurely) but I don't think it is enough of a reason to prevent a merge

**Review by encima — APPROVED** [review]

**Comment by rikonen** [comment]
@ivan-savciuc are you going to check this one?

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**